agent/php_agent.h

```diff
@@ -863,8 +880,28 @@ extern void nr_php_txn_add_code_level_metrics(
  *           or NULL if the zend_execute_data is invalid.
  *
  */
-extern const char* nr_php_zend_execute_data_function_name(
-    const zend_execute_data* execute_data);
+static inline const char* nr_php_zend_execute_data_function_name(
+    const zend_execute_data* execute_data) {
+  zend_string* function_name = NULL;
+
+  if ((NULL == execute_data) || (NULL == execute_data->func)) {
+    return NULL;
+  }
+
+  function_name = execute_data->func->common.function_name;
+
+  if ((NULL == function_name)
+      && (ZEND_USER_FUNCTION == execute_data->func->type)) {
+    /*
+     * This is the case of a filename being called so there is no function name.
+     * It is broken out separately here in case we need to do something special
+     * with it in the future.
+     */
+    return NULL;
```

Comment:
For performance gains in the present, do we want to comment out this entire block?